apis/core/v1beta1/common/common.go

```diff
@@ -12,6 +16,36 @@ const (
 	SyncProviderFlagdProxy SyncProviderType = "flagd-proxy"
 )
 
+const (
+	ManagementPortEnvVar        string = "MANAGEMENT_PORT"
+	PortEnvVar                  string = "PORT"
+	HostEnvVar                  string = "HOST"
+	TLSEnvVar                   string = "TLS"
+	SocketPathEnvVar            string = "SOCKET_PATH"
+	OfflineFlagSourcePathEnvVar string = "OFFLINE_FLAG_SOURCE_PATH"
+	SelectorEnvVar              string = "SOURCE_SELECTOR"
+	CacheEnvVar                 string = "CACHE"
+	CacheMaxSizeEnvVar          string = "MAX_CACHE_SIZE"
+	ResolverEnvVar              string = "RESOLVER"
```

Comment:
This var (`FLAGD_RESOLVER`) needs to be set, especially in the in-process mode, but I don't see it used. cc @beeme1mr am I missing something?

Reply:
On a related note, it looks like in Java we don't define this var: https://github.com/open-feature/java-sdk-contrib/blob/703830d5247b4a52a0b0767bf2d13a7e93582d00/providers/flagd/README.md?plain=1#L88

We need to do this to support auto-config of in-process flagd Java via OFO.

Reply:
And looks like we have a bug (wrong var name) in dotnet: https://github.com/open-feature/dotnet-sdk-contrib/blob/38a59f01b4c740ddcfb69b68c8b79fb169e06ad4/src/OpenFeature.Contrib.Providers.Flagd/FlagdConfig.cs#L39

Reply:
Fix for c#: https://github.com/open-feature/dotnet-sdk-contrib/pull/206

Reply:
adapted